src/Microsoft.Data.SqlClient/tests/ManualTests/Extensions/StreamExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿#if NETFRAMEWORK
+
+namespace System.IO
+{
+    // Helpers to read/write Span/Memory<byte> to Stream before netstandard 2.1
+    internal static class StreamExtensions
```

Comment:
This class shouldn't be necessary - the internals of the netfx project should be available to the test projects. If they aren't, we need to fix that rather than duplicate this code.

Reply:
How would you do that, typically? Using this attribute: https://learn.microsoft.com/en-us/dotnet/fundamentals/runtime-libraries/system-runtime-compilerservices-internalsvisibletoattribute ?